type-overrides/src/lib.rs

```diff
@@ -0,0 +1,55 @@
+///
+/// This lib contains both standard imports and imports shuttle.
+/// Shuttle is a Rust crate that facilitates multithreaded testing. It has its own scheduler
+/// and can efficiently detect bugs in concurrent code. The downside is that we need to replace
+/// all imports by those from Shuttle.
+///
+/// Instead of importing from std, rand, and so on, import the following from solana-type-override,
+/// and include the 'shuttle-test' feature in your crate to use shuttle.
+pub mod executor {
+    #[cfg(not(feature = "shuttle-test"))]
+    pub use futures::executor::*;
+    #[cfg(feature = "shuttle-test")]
+    pub use shuttle::future::*;
+}
+
+pub mod hint {
+    #[cfg(feature = "shuttle-test")]
+    pub use shuttle::hint::*;
+    #[cfg(not(feature = "shuttle-test"))]
+    pub use std::hint::*;
+}
+
+pub mod lazy_static {
+    #[cfg(not(feature = "shuttle-test"))]
+    pub use lazy_static::*;
+    #[cfg(feature = "shuttle-test")]
+    pub use shuttle::lazy_static::*;
+}
+
+pub mod rand {
+    #[cfg(not(feature = "shuttle-test"))]
+    pub use rand::{thread_rng, Rng, RngCore};
+    #[cfg(feature = "shuttle-test")]
+    pub use shuttle::rand::{thread_rng, Rng, RngCore};
+}
+
+pub mod rand_original {
+    pub use rand::{
```

Comment:
I think these should be exported inside rand not rand_original? Otherwise we still have the same problem

Reply:
I got confused with your last message asking for a type alias. I thought you didn't want the other types to be exported under the same prefix.